### DISCUSSION TITLE
Ability to create a "Dashboard" of several specific patch locations in the Time Tracking plugin that all show together in the same category

### DISCUSSION THREAD

**Discussion by Rockin132** [runelite/runelite#18207] [Ideas]
Instead of having to click through all the different categories (i.e. Bush Patches, Tree Patches, Fruit Patches) when I have multiple different things planted and growing -- I would really like the ability to have a single panel a bit like the overview panel where I can select, for example, 3 Fruit Tree Locations, 6 Herb Locations, and 4 Tree Patches and have them all show together in a list inside the same category with their timers.

**Comment by cdfisher** [comment]
This is effectively a duplicate of #16268
